core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -188,6 +188,20 @@ public void setCompression(String compressor) {
     // noop
   }
 
+  /**
+   * Defers flushing the messages written using {@link #sendMessage(Object)} to the
+   * transport until the write buffer is full. The amount of data buffered while corked is
+   * dependent on the transport implementation so users of this API should not make strong
+   * assumptions about how long data is buffered while corked.
+   *
+   * <p>Corking can provide throughput gains by allowing a transport to perform fewer writes to
+   * the underlying network.
+   */
```

Comment:
Reading the javadoc I'm uncertain the following:
Is this allowed to call this method multiple times without closing the previous one before calling next one?
If allowed, will it return a different or the same instance? If returning a different instance, what is expected behavior?

Reply:
added a paragraph that describes the behavior